bookkeeper-server/src/main/java/org/apache/bookkeeper/util/collections/ConcurrentLongHashSet.java

```diff
@@ -339,15 +344,15 @@ public void forEach(ConsumerLong processor) {
                 }
 
                 // Go through all the buckets for this section
-                for (int bucket = 0; bucket < table.length; bucket++) {
-                    long storedItem = table[bucket];
+                for (long l : table) {
```

Comment:
Thi s is creating an Iterator, can we save this allocation? 
Also the JVM is very good in dealing with counted loops